db/migrate/20160913195129_move_repo_data_from_database_to_settings.rb

```diff
@@ -30,7 +30,7 @@ def down
   end
 
   def my_region
-    MiqRegion.find_by_region(ApplicationRecord.my_region_number)
+    MiqRegion.find_by_region(ArRegion.anonymous_class_with_ar_region.my_region_number)
```

Comment:
Since you're in here, can you fix this to `find_by(:region => ...)`... or do it in a followup PR if this is about to pass travis?